### PR TITLE
media-gfx/blender: Remove redundant dep flags and bump embree package.

### PR DESCRIPTION
**Pull request by DarkDefender**
I noticed that the opensubdiv dependency incorrectly pulls in CUDA (and opencl) even if Blender doesn't use these at all.
(Only the CPU and OpenGL backend is used by Blender)

I also bumped the embree package while I was at it.

### CODE REVIEW

File: media-libs/embree/embree-3.13.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Copyright 1999-2021 Gentoo Authors
```
Comment (juippis) [inline media-libs/embree/embree-3.13.4.ebuild:1]:
```suggestion
# Copyright 1999-2022 Gentoo Authors
```

File: media-libs/embree/embree-3.13.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline media-libs/embree/embree-3.13.4.ebuild:4]:
```suggestion
EAPI=8
```

File: media-libs/embree/embree-3.13.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+LICENSE="Apache-2.0"
+SLOT="3"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+X86_CPU_FLAGS=( sse2:sse2 sse4_2:sse4_2 avx:avx avx2:avx2 avx512dq:avx512dq )
+CPU_FLAGS=( cpu_flags_arm_neon ${X86_CPU_FLAGS[@]/#/cpu_flags_x86_} )
+IUSE="+compact-polys ispc +raymask ssp +tbb tutorial ${CPU_FLAGS[@]%:*}"
+# Let's be explicit here even though we could simplify it.
+REQUIRED_USE="amd64? ( cpu_flags_x86_sse2 ) x86? ( cpu_flags_x86_sse2 )"
+
+RESTRICT="mirror"
```
Comment (juippis) [inline media-libs/embree/embree-3.13.4.ebuild:21]: What, why? The listed license shouldn't forbid this.

Comment (DarkDefender) [reply media-libs/embree/embree-3.13.4.ebuild:21]: I don't know.
It was like this in the older ebuilds.
I just did a version bump and copied an older ebuild.

**Comment by DarkDefender** [comment]
@juippis I guess I can just remove the `RESTRICT="mirror"` because I don't find anything that should prevent us from rehosting the files either.
Probably was something that just slipped into the build at some point.

**Review by juippis — COMMENTED** [review]
```
[ERROR] >>> Not all runs were successful.
atom: =media-libs/embree-3.13.4, USE flags: 'compact-polys -ispc raymask -ssp tbb tutorial'
atom: =media-libs/embree-3.13.4, USE flags: 'compact-polys ispc raymask ssp -tbb tutorial'
atom: =media-libs/embree-3.13.4, USE flags: '-compact-polys -ispc -raymask -ssp -tbb tutorial'
atom: =media-libs/embree-3.13.4, USE flags: '-compact-polys -ispc raymask -ssp tbb tutorial'
atom: =media-libs/embree-3.13.4, USE flags: '-compact-polys ispc -raymask -ssp -tbb tutorial'
```
It looks like `+tutorial` use flag is problematic for embree. Can you try? Seems like this is the issue:

```
In file included from /usr/include/OpenImageIO/dassert.h:12,
                 from /usr/include/OpenImageIO/span.h:17,
                 from /usr/include/OpenImageIO/imageio.h:31,
                 from /var/tmp/portage/media-libs/embree-3.13.4/work/embree-3.13.4/tutorials/common/   image/oiio.cpp:11:
/usr/include/OpenImageIO/platform.h:97:6: error: #error "This version of OIIO is meant to work only    with C++14 and above"
   97 | #    error "This version of OIIO is meant to work only with C++14 and above"
      |      ^~~~~
```

**Comment by DarkDefender** [comment]
Fixed!

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**